### ISSUE TITLE
Not compatible with 2020.3

### ISSUE THREAD

**Issue by kgignatyev** [will7200/casbin-idea-plugin#3]
Just upgraded to 2020.3 and it marked plugin as not compatible.

IntelliJ IDEA 2020.3 (Ultimate Edition)
Build #IU-203.5981.155, built on November 30, 2020
Runtime version: 11.0.9+11-b1145.21 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 10.15.7
GC: ParNew, ConcurrentMarkSweep
Memory: 1981M
Cores: 8

**Comment by kgignatyev** [comment]
tried to compile with new Idea but got a lot of 

CasbinParserTest > testabac_rule_with_domains_model FAILED
    java.lang.NoClassDefFoundError: Could not initialize class com.intellij.testFramework.TemporaryDirectory
        at com.intellij.testFramework.UsefulTestCase.createGlobalTempDirectory(UsefulTestCase.java:258)
        at com.intellij.testFramework.UsefulTestCase.setupTempDir(UsefulTestCase.java:243)

**Comment by hsluoyz** [comment]
@will7200

**Comment by will7200** [comment]
I'll take a look. I haven't tested for the the 2020.3 versions

**Comment by kgignatyev** [comment]
Much appreciated, really useful plugin, thank you @will7200

**Comment by will7200** [comment]
I looked into it and I believe this is due to IDE config difference. I marked the test resources via Intellij. Not sure why it is not picking it up since it is defined in the build.gradle.kts file. But none the less that are some other issues that will need my attention.
1. 2020.3 Intellij Products now using JDK 11. 
2. With the newer version, the casbin editor is registering some errors that I need to futher dive into.
